### PR TITLE
Add 0 at the end of file read result in AssembleSpirv

### PR DESCRIPTION
**Pull request by linqun**

### CODE REVIEW

File: tool/amdllpc.cpp [hunk 1]
```diff
@@ -710,6 +710,8 @@ static Result AssembleSpirv(
         memset(pSpvText, 0, textSize + 1);
 
         size_t realSize = fread(pSpvText, 1, textSize, pInFile);
+        pSpvText[realSize] = 0;
```
Comment (amdrexu) [inline tool/amdllpc.cpp]: = 0 -> = '\0'

**Comment by amdvlk-admin** [comment]
### Test summary for commit 95cd4da1b2e8a6aa8bdb8caf0b4610ad9a78cff8
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev 9a1f99f049f3b417242dc7c09b86c52fd7d9ceaf</li>
<li>SPVGEN: dev 64013f150c2d41965a3d36fa159b55fba52b727a</li>
<li>XGL: dev 4d91a4045078cf6884ee05007eac9eec2b881dbb</li>
<li>LLVM: amd-vulkan-dev 4305f776f1207189667a383e22661ccd1e99cd25</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 893/893 (100.0%)</li>
<li>Failed: 0/893 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Vega20</summary><ul>

<li>Passed: 294147/294148 (100.0%)</li>
<li><details><summary>Failed: 1/294148 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.image.swapchain_mutable.xlib.2d.r8g8b8a8_unorm_r8g8b8a8_snorm_clear_copy_format_list
Stack trace: Failed to open display: '' at tcuLnxX11.cpp:67
```
</p></details></li>


</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 3546299350182b6994876d2b31aa8cab45e022a1
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev 9a1f99f049f3b417242dc7c09b86c52fd7d9ceaf</li>
<li>SPVGEN: dev 64013f150c2d41965a3d36fa159b55fba52b727a</li>
<li>XGL: dev 4d91a4045078cf6884ee05007eac9eec2b881dbb</li>
<li>LLVM: amd-vulkan-dev 4305f776f1207189667a383e22661ccd1e99cd25</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 893/893 (100.0%)</li>
<li>Failed: 0/893 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Vega20</summary><ul>

<li>Passed: 294147/294148 (100.0%)</li>
<li><details><summary>Failed: 1/294148 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.image.swapchain_mutable.xlib.2d.r8g8b8a8_unorm_r8g8b8a8_snorm_clear_copy_format_list
Stack trace: Failed to open display: '' at tcuLnxX11.cpp:67
```
</p></details></li>


</ul></details>


</ul></details>

**Comment by JacobHeAMD** [comment]
retest this please

**Comment by JaxLinAMD** [comment]
retest this please

**Comment by JacobHeAMD** [comment]
Hi @linqun ,  please rebase your commit to the latest dev, it will cause Vega10 agent of Jenkins hang otherwise.

**Comment by JaxLinAMD** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit 3546299350182b6994876d2b31aa8cab45e022a1
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev 9a1f99f049f3b417242dc7c09b86c52fd7d9ceaf</li>
<li>SPVGEN: dev 64013f150c2d41965a3d36fa159b55fba52b727a</li>
<li>XGL: dev 4d91a4045078cf6884ee05007eac9eec2b881dbb</li>
<li>LLVM: amd-vulkan-dev 4305f776f1207189667a383e22661ccd1e99cd25</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 893/893 (100.0%)</li>
<li>Failed: 0/893 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 366444/366444 (100.0%)</li>
<li>Failed: 0/366444 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**